src/module/implements/implementBenefits/lantern.js

```diff
@@ -93,73 +59,32 @@ class Lantern extends Implement {
         option: "lantern-implement-lit",
         toggleable: true,
       },
-      {
-        domain: "all",
-        key: "RollOption",
-        label: "Target Within Lantern Bright Light",
-        option: "target-in-lantern-bright-light",
-        toggleable: true,
-        predicate: ["lantern-implement-lit"],
-        hideIfDisabled: true,
-      },
       {
```

Comment:
I believe that these can be added into the aura if the aura has `affects:{includesSelf:true}`, no? Is there a benefit with keeping it segregated? People might be confused if they don't see the effect on themself as well as their allies.